engine/language-server/src/main/java/org/enso/languageserver/boot/resource/RepoInitialization.java

```diff
@@ -63,7 +63,7 @@ public CompletableFuture<Void> init() {
     return initSuggestionsRepo()
         .whenCompleteAsync(
             (res, err) -> {
-              if (err == null && res != null) {
+              if (err == null) {
```

Comment:
`res` is always `null` because it is the only instance of type `Void`

Reply:
I assume this is unrelated to `Ydoc` server.